symengine/tests/eval/test_eval_mpfr.cpp

```diff
@@ -53,5 +54,15 @@ TEST_CASE("precision: eval_mpfr", "[eval_mpfr]")
     REQUIRE(mpfr_cmp_d(a, 0.00000000490153) == 1);
     REQUIRE(mpfr_cmp_d(a, 0.00000000490154) == -1);
 
+    mpfr_init2(a, 100);
```

Comment:
This should be `mpfr_set_prec(a, 100);`. Initializing an already initialized `mpfr_t` may lead to memory leaks.